### ISSUE TITLE
Other Surveys

### ISSUE THREAD

**Issue by profxj** [pyigm/igmspec#9]
Please add to this list:

- HST STIS/GHRS from Cooksey+10
- HST COS-Halos, COS-Dwarfs
- Gabor's HeII survey
- MagE LLS survey (Fumagalli+ 15);  overlaps with HD-LLS
- H100 (Neeleman+13)
- ESI DLAs  (Rafelski)
- GRBs galore
- QPQ (eventually)
- HIRES dataset from UCSD (Pro+07)

**Comment by astronomeara** [comment]
-WFC3/ACS qsos from O'Meara HST programs

**Comment by astronomeara** [comment]
- Lick QSOs?

**Comment by astronomeara** [comment]
- Ribaudo HST sample

**Comment by profxj** [comment]
UVQS_DR1

**Comment by profxj** [comment]
- 2dF
- QSO photo z catalogs

**Comment by astronomeara** [comment]
KODIAQ DR2 at end of summer/early fall.

**Comment by astronomeara** [comment]
What are our thoughts on absorber catalogues?

**Comment by profxj** [comment]
All in favor.  Will be a separate database.  

Got a name for it?

> On Jul 14, 2016, at 5:30 AM, astronomeara <notifications@github.com> wrote:
> 
> What are our thoughts on absorber catalogues?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/pyigm/igmspec/issues/9#issuecomment-232651682>, or mute the thread <https://github.com/notifications/unsubscribe/AD0BjURJgSaZMk_VtRD9ZMBKqBaPkNOYks5qVivjgaJpZM4I5Dry>.
> 
> !DSPAM:826,578785ed257521665020033!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)

**Comment by astronomeara** [comment]
igmabs

J

**Comment by profxj** [comment]
We shall make it so.
For what it’s worth, pyigm already has *a lot* of this in it.
LLS, DLAs and CGM soon.

Biggest challenge will be data format.  I’ve been using JSON files
and those probably don’t pack into hdf5 so nicely.

> On Jul 14, 2016, at 5:36 AM, astronomeara <notifications@github.com> wrote:
> 
> igmabs
> 
> J
> 
> On Jul 14, 2016, at 8:35 AM, J. Xavier Prochaska <notifications@github.com> wrote:
> 
> > All in favor. Will be a separate database. 
> > 
> > Got a name for it?
> > 
> > > On Jul 14, 2016, at 5:30 AM, astronomeara <notifications@github.com> wrote:
> > > 
> > > What are our thoughts on absorber catalogues?
> > > 
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub <https://github.com/pyigm/igmspec/issues/9#issuecomment-232651682>, or mute the thread <https://github.com/notifications/unsubscribe/AD0BjURJgSaZMk_VtRD9ZMBKqBaPkNOYks5qVivjgaJpZM4I5Dry>.
> > > 
> > > 
> > 
> > ----------------------------------------------
> > J. Xavier Prochaska
> > UCO/Lick Observatory
> > 1156 High St.
> > UC Santa Cruz
> > Santa Cruz, CA 95064
> > 
> > http://www.ucolick.org/~xavier/
> > http://imps.ucolick.org/
> > http://www.qpqsurvey.org/
> > http://k1dm3.ucolick.org/
> > http://www.ucolick.org/~xavier/IDL/index.html
> > https://github.com/pyigm/pyigm
> > https://github.com/linetools/linetools
> > 
> > 831-459-2135 (Direct)
> > 831-459-2991 (UCO/Lick Main)
> > 831-459-5244 (Fax)
> > 
> > 
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> > 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/pyigm/igmspec/issues/9#issuecomment-232652946>, or mute the thread <https://github.com/notifications/unsubscribe/AD0BjemQGqPKoyNCP_v1yXg8t4Kpg19Cks5qVi07gaJpZM4I5Dry>.
> 
> !DSPAM:826,5787873c25889274382295!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)

**Comment by astronomeara** [comment]
Right.  I was thinking gonzo things like integrating Cooksey’s MgII or CIV searches in SDSS, etc….

J
